main_2024.py

```diff
@@ -140,21 +140,46 @@ def main() -> int:
         QUEUE_MAX_SIZE,
     )
 
-    video_input_manager = worker_manager.WorkerManager()
-    video_input_manager.create_workers(
+    result, camera_intrinsics = camera_properties.CameraIntrinsics.create(
+        GEOLOCATION_RESOLUTION_X,
+        GEOLOCATION_RESOLUTION_Y,
+        GEOLOCATION_FOV_X,
+        GEOLOCATION_FOV_Y,
+    )
+    if not result:
+        print("Error creating camera intrinsics")
+        return -1
+
+    result, camera_extrinsics = camera_properties.CameraDroneExtrinsics.create(
+        (
+            GEOLOCATION_CAMERA_POSITION_X,
+            GEOLOCATION_CAMERA_POSITION_Y,
+            GEOLOCATION_CAMERA_POSITION_Z,
+        ),
+        (
+            GEOLOCATION_CAMERA_ORIENTATION_YAW,
+            GEOLOCATION_CAMERA_ORIENTATION_PITCH,
+            GEOLOCATION_CAMERA_ORIENTATION_ROLL,
+        ),
+    )
+    if not result:
+        print("Error creating camera extrinsics")
+        return -1
+
+    # Worker arguments
+    video_input_worker_args = (
         1,
         video_input_worker.video_input_worker,
         (
             VIDEO_INPUT_CAMERA_NAME,
             VIDEO_INPUT_WORKER_PERIOD,
             VIDEO_INPUT_SAVE_PREFIX,
-            video_input_to_detect_target_queue,
-            controller,
         ),
+        [],
+        [video_input_to_detect_target_queue],
+        controller,
```

Comment:
we should explicitly state the params since this gets confusing
```
class_args = ( ... ),
input_queues = [],
output_queues = [video_input_to_detect_target_queue],
...
```

Reply:
Not possible as discussed